lib/postgres.js

```diff
@@ -249,6 +250,35 @@ class Postgres {
                 }
             }
 
+            const keychains = item.keychains.length > 0 ? item.keychains.map((s) => {
+                const res = {s: s.slot, i: s.sticker_id};
+                if (s.wear) {
+                    res.w = s.wear;
+                }
+                if (s.scale) {
+                    res.scale = s.scale;
```

Comment:
Since JSONB doesn't have meaningful compression in Postgres -- we should try to constraint our key lengths.

`scale` -> `s`, `tint_id` -> `t`, `pattern` -> `p`

Reply:
Thanks for your suggestion!  Yes, you're right.  The 's' conflicts with the existing 'slot', so I renamed 'scale' to 'sc'.